content/en/docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -110,86 +51,57 @@ follow
 
 See [Installing kubeadm](/docs/setup/independent/install-kubeadm/).
 
-**Note:** If you already have kubeadm installed, you should do a `apt-get update &&
+**Note:** If you already have kubeadm installed, run `apt-get update &&
 apt-get upgrade` or `yum update` to get the latest version of kubeadm.
 
-
 The kubelet is now restarting every few seconds, as it waits in a crashloop for
-kubeadm to tell it what to do. This crashloop is expected and normal, please
-proceed with the next step and the kubelet will start running normally.
+kubeadm to tell it what to do. This crashloop is expected and normal. 
+After you initialize your master, the kubelet starts running normally.
 
 ### Initializing your master
 
 The master is the machine where the control plane components run, including
 etcd (the cluster database) and the API server (which the kubectl CLI
 communicates with).
 
-To initialize the master, first choose the pod network plugin you want and check if it requires any parameters to be passed to kubeadm while initializing the cluster. Pick one of the machines you previously installed
-kubeadm on, and run:
+1. Choose a pod network add-on, and verify whether it requires any arguments to 
+be passed to kubeadm initialization. Depending on which
+third-party provider you choose, you might need to set the `--pod-network-cidr` to
+a provider-specific value. See [Installing a pod network add-on](#pod-network).
+1. (Optional) Unless otherwise specified, kubeadm uses the network interface associated 
+with the default gateway to advertise the master's IP. To use a different 
+network interface, specify the `--apiserver-advertise-address=<ip-address>` argument 
+to `kubeadm init`. To deploy an IPv6 Kubernetes cluster using IPv6 addressing, you 
+must specify an IPv6 address, for example `--apiserver-advertise-address=fd00::101`
+
+Now run:
 
 ```bash
-kubeadm init
+kubeadm init <args> 
```

Comment:
I don't think this code block is very informative. Maybe give an example real-world command. Is it possible there won't be any args? What does the output look like?